modules/search-pipeline-common/src/test/java/org/opensearch/search/pipeline/common/RenameResponseProcessorTests.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a.java
+ * compatible open source license.
+ */
+
+package org.opensearch.search.pipeline.common;
+
+import org.apache.lucene.search.TotalHits;
+import org.opensearch.OpenSearchParseException;
+import org.opensearch.action.search.SearchRequest;
+import org.opensearch.action.search.SearchResponse;
+import org.opensearch.action.search.SearchResponseSections;
+import org.opensearch.common.bytes.BytesArray;
+import org.opensearch.common.document.DocumentField;
+import org.opensearch.index.query.QueryBuilder;
+import org.opensearch.index.query.TermQueryBuilder;
+import org.opensearch.ingest.RandomDocumentPicks;
+import org.opensearch.search.SearchHit;
+import org.opensearch.search.SearchHits;
+import org.opensearch.search.builder.SearchSourceBuilder;
+import org.opensearch.test.AbstractBuilderTestCase;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RenameResponseProcessorTests extends AbstractBuilderTestCase {
```

Comment:
Oh -- a minor tweak: this can (probably?) extend `OpenSearchTestCase`.

I only needed to extend `AbstractBuilderTestCase` in `FilterQueryResquestProcessorTests` because I needed a `NamedXContentRegistry` instance in order to parse queries. Most processor tests should (hopefully) be able to extend the more lightweight `OpenSearchTestCase`.

Reply:
Resolved in [37aeb87](https://github.com/opensearch-project/OpenSearch/pull/7377/commits/37aeb8734ecb7dae90d554fecab483513f741f25)